lib/common/schemas.c

```diff
@@ -852,6 +843,22 @@ pcmk__validate_xml(xmlNode *xml_blob, const char *validation,
     return false;
 }
 
+/*!
+ * \internal
+ * \brief Validate XML using its configured schema (and send errors to logs)
+ *
+ * \param[in] xml  XML to validate
+ *
+ * \return TRUE if XML validates, otherwise FALSE
```

Comment:
I don't know how consistent we're being in our docstrings, but it actually returns `true` or `false`.

Reply:
whoops, thanks